chips/atmega328p-hal/src/lib.rs

```diff
@@ -178,9 +179,10 @@ pub mod spi {
                 sclk: portc::PC1,
                 mosi: porte::PE3,
                 miso: portc::PC0,
-                cs: portc::PE2,
+                cs: porte::PE2,
```

Comment:
Nice catch, I wonder how this ever worked ...

Reply:
I'm also surprised. Thanks for fixing this.